src/atoms/Link/Link.js

```diff
@@ -46,25 +46,24 @@ export default function Link({
   //If it does, append it to the end our current pages url for "to".
   const isAnchor = /^[#]/.test(linkHref);
 
-  if (isAnchor && typeof window !== "undefined") {
+  if (isAnchor && typeof window !== 'undefined') {
     return (
       <GatsbyLink
         to={`${window.location.pathname}${linkHref}`}
-        sx={{ variant: "styles.a" }}
+        sx={{ variant: 'styles.a' }}
         onClick={() => {
-          //TODO(Rejon): See how marketing team wants to handle anchor links.
-          // if (onClick !== null && onClick !== undefined) {
-          //   onClick();
-          // }
-          // const eventProps = Object.assign(
-          //   {
-          //     category: "Internal Link",
-          //     action: "Click",
-          //     label: linkHref,
-          //   },
-          //   gaProps
-          // );
-          // trackCustomEvent(eventProps);
+          if (onClick !== null && onClick !== undefined) {
```

Comment:
we can simply do `if (onClick) {...}`. Although I don't think we should check for `onClick` to be defined since in the type definition this prop is not optional, check line 25 (`onClick: () => void`) that means we always receive prop onClick, otherwise it would be (`onClick?: () => void`). So, that means if this component is called somewhere in code without this prop flow types check won't pass.